Resources/views/Form/fields.html.twig

```diff
@@ -3,45 +3,41 @@
     {{- block('form_row') }}
 {%- endblock %}
 
-{% block vich_file_widget %}
-    {% filter spaceless %}
-        <div class="vich-file">
-            {{ form_widget(form.file) }}
-            {% if form.delete is defined %}
-                {{ form_row(form.delete) }}
-            {% endif %}
+{%- block vich_file_widget -%}
```

Comment:
I guess that starting space removal is useless when opening block.
Same applies for ending space removal when closing blocks.

Reply:
I suppose it’s not strictly necessary, however, at least in my projects, I prefer to use space removal on all tags for consistency’s sake, and to reduce the likelihood of surprises down the road.

Reply:
However I can trim the spaceless tags to the minimum necessary, if you want.

Reply:
I'll appreciate consistency.

So, I see 3 possible options:
- space removal everywhere
- space removal everywhere except where blatantly unnecessary (as mentioned before)
- keep spaceless filter

Reply:
The changes I committed do space removal everywhere; the only tags that don’t were the open/close tags for vich_file_row, vich_image_row, and vich_file_label, which didn’t strictly need them, and I left them off to reduce the diff size. So, I’ll change them to use space removal so the whole file is consistent.